server/modules/slackIntegration.js

```diff
@@ -0,0 +1,92 @@
+import { WebClient } from '@slack/client';
+import { findPartnerById } from './allocations';
+import makeChannelNames from '../helpers/slackHelpers';
+import slackMocks from '../../test/mocks/slack';
+import allocationsMocks from '../../test/mocks/allocations';
+
+
+const { SLACK_TOKEN } = process.env;
+const slackClient = new WebClient(SLACK_TOKEN);
+
+
+/**
+ * @function createChannel
+ * @desc Create slack channel
+ *
+ * @param {string} name - Name of the channel
+ * @returns {Object} - An object containing the newly created channel
+ */
+export const createChannel = async (name) => {
+  try {
+    const { create } = slackClient.groups;
+    const createResult = await create({ name });
+
+    return createResult;
+  } catch (error) {
+    return error;
+  }
+};
+
+
+/**
+ * @function createForPartner
+ * @desc Create slack channels for a partner engagement
+ *
+ * @param {string} partnerId - ID of the partner
+ * @returns {Object} - An object containing details of the created channels
+ */
+export const createPartnerChannels = async (partnerId) => {
+  try {
+    let partner;
+
+    if (process.env.NODE_ENV === 'test') {
```

Comment:
This should be controlled from the test case and not written into the body of the function